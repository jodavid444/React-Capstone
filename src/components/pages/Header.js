import React from 'react';
import logo from '../../images/123123.png';

function Header() {
  return (
    <header>
      <img src={logo} alt="logo" className="logo" />
      <h1>Grub Hub Foods Tract </h1>
      <i className="fa-solid fa-microphone" />
    </header>
  );
}

export default Header;
