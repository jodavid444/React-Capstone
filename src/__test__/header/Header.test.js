import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Header from '../../components/pages/Header';

jest.mock('react-redux');

describe('Test component', () => {
  test('Test Header', () => {
    const tree = render(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
  
});
