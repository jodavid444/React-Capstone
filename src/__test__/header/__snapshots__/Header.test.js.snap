// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component Test Header 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header>
        <img
          alt="logo"
          class="logo"
          src="123123.png"
        />
        <h1>
          Chef's Masterpieces
        </h1>
        <i
          class="fa-solid fa-microphone"
        />
      </header>
    </div>
  </body>,
  "container": <div>
    <header>
      <img
        alt="logo"
        class="logo"
        src="123123.png"
      />
      <h1>
        Chef's Masterpieces
      </h1>
      <i
        class="fa-solid fa-microphone"
      />
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
